### PR TITLE
Provide an icon to the upgrade link so that people can upgrade their login token to add things to CI.

### LINKED ISSUE #37: Need support in the UI for /login/upgrade
A new feature to grant additional scopes to oauth2 tokens, /login/upgrade (from uisvc) will allow the user to do that.

Since documenting a url further than the swagger API seems like folly, we should really make this a button or something in the UI (maybe near the logout button?) so we can trigger it from there instead of editing URLs in our browsers.

### PR DESCRIPTION
**Pull request by erikh**
closes #37

### CODE REVIEW

**Comment by erikh** [comment]
added some tooltips and also made a patch to resolve #32 

PTAL @capoferro

File: src/components/repolist/RepoList.js [hunk 1]
```diff
@@ -128,7 +137,20 @@ class RepoList extends BaseComponent {
...
+              {this.props.username}
+              <Tooltip
+                title="Upgrade user account & give additional privileges to tinyCI"
+                aria-label="Upgrade user account & give additional privileges to tinyCI"
+              >
+                <IconButton
+                  style={{ color: "white" }}
+                  href="/uisvc/login/upgrade"
+                >
+                  <Icon fontSize="small">arrow_upward</Icon>
```
Comment (capoferro) [inline src/components/repolist/RepoList.js]: Can this icon be imported directly like the other ones?

**Comment by capoferro** [comment]
One minor comment, otherwise looks good.

File: src/components/repolist/RepoList.js (hunk 1 above)
Comment (capoferro) [reply src/components/repolist/RepoList.js]: I'm not sure it's technically better, but that's how I've been doing it:

`import ViewListIcon from "@material-ui/icons/ViewList"`

Comment (erikh) [reply src/components/repolist/RepoList.js]: ok, i'll see if that's possible.

**Pull request merged**